.github/workflows/codeql.yml

```
@@ -27,6 +27,12 @@ jobs:
         with:
           languages: python, javascript
           queries: security-and-quality
+          paths-ignore: |
```

actionlint (1.7.11)

[error] input "paths-ignore" is not defined in action "github/codeql-action/init@v4". available inputs are "analysis-kinds", "build-mode", "config", "config-file", "db-location", "debug", "debug-artifact-name", "debug-database-name", "dependency-caching", "external-repository-token", "languages", "matrix", "packs", "quality-queries", "queries", "ram", "registries", "setup-python-dependencies", "source-root", "threads", "token", "tools", "trap-caching"

(action)